process_trees.py

```diff
@@ -87,8 +69,16 @@ def get_output(trees):
 
     tree = all_trees[2]
 
-    tree_indices = get_tree_indices(tree)
+    tree_indices = assign_indices_to_nodes(tree)
     adj_mat = get_adj_matrix(tree, tree_indices)
     texts = get_tree_text(tree)
     embedding, offsets = get_tree_text_embedding(texts)
-
+    outputs = get_output(tree)
+
+    model_data = {
+        "adj_matrix": adj_mat,
+        "embedding": embedding,
+        "offsets": offsets,
+        "outputs": outputs
+    }
```

Comment:
So `tree_indices` is not necessary context here because everything has been translated to ordered lists, and the `i`th element in each of the `model_data` elements corresponds to the same node, right?

Reply:
Yes, tree_indices is just a way to order the nodes to create the adjacency matrix.